secstatecheck

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+FULL_PATH_TO_SCRIPT="$(realpath "$0")"
+SCRIPT_DIRECTORY="$(dirname "$FULL_PATH_TO_SCRIPT")"
+
+if [ ! -f "$SCRIPT_DIRECTORY/.ENV" ]; then
+    echo "$SCRIPT_DIRECTORY/.ENV does not exist."
+    echo "ENV.example is template to use"
+    exit 1
+fi
+source "$SCRIPT_DIRECTORY"/.ENV
+PID=$$
+mkdir -p "$DIR"
+LOG="$DIR/seccheck.${PID}.log"
```

Comment:
Use another log file so it's easy to see different checks

Reply:
Fixed